### PR TITLE
Add PyTorch/Python test variations and fix test to pass on them

### PR DESCRIPTION
**Pull request by kmaehashi**
Add following test variations:

* Python 3.5 + PyTorch 1.5
* Python 3.6 + PyTorch 1.6
* Python 3.8 + PyTorch 1.7

TODO:
- [x] We have to fix ONNX tests. #84 changed ONNX tests to pass on PyTorch 1.7, but it fails with PyTorch 1.5/1.6.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Test passed! Please ignore `pytorch-pfn-extras` failure as this variant will be removed once this PR got merged.

File: tests/pytorch_pfn_extras_tests/onnx/test_export_testcase.py [hunk 1]
```diff
@@ -19,6 +19,9 @@
 
 output_dir = 'out'
 
+# PyTorch (major, minor), e.g., (1, 7)
+torch_version = tuple(int(x) for x in torch.__version__.split('.')[0:2])
```
Comment (asi1024) [inline tests/pytorch_pfn_extras_tests/onnx/test_export_testcase.py:23]: Could you fix it like as https://github.com/pfnet/pytorch-pfn-extras/pull/96?

**Comment by asi1024** [comment]
@kmaehashi Could you check test failures?

**Comment by kmaehashi** [comment]
Test passed!

**Pull request merged**